src/optlang/gurobi_interface.py

```diff
@@ -36,7 +36,7 @@
 
 try:
     version = gurobipy.gurobi.version()
-    if not (version[0] >= 9 and version[1] >= 5):
+    if not version[:2] >= (9, 5):
```

Comment:
Would it be more straight forward to say
```suggestion
    if version[:2] < (9, 5):
```
?